docs/user-guide/compute-resources.md

```diff
@@ -91,10 +91,16 @@ runner (Docker or rkt).
 
 When using Docker:
 
-- The `spec.container[].resources.limits.cpu` is multiplied by 1024, converted to an integer, and
-  used as the value of the [`--cpu-shares`](
+- The `spec.container[].resources.requests.cpu` is converted to its millicore value,
+  multipled by 1024, and then divided by 1000, and used as the value of the [`--cpu-shares`](
```

Comment:
Conceptually the way the comment above the code makes more sense to me:
// Conceptually (milliCPU / milliCPUToCPU) * sharesPerCPU, but factored to improve rounding.

aka Converted to cores (potentially fractional) then multiplied by 1024